.travis.yml

```diff
@@ -12,4 +13,4 @@ script:
   - ./go.test.sh
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+- bash <(curl -s https://codecov.io/bash)
```

Comment:
why change here ?

Reply:
not intentional

Reply:
please be more careful, seems you fixed but there is still unnecessary whitespace change

Reply:
I am but it seems when I save it in vscode something goes wrong. But will be much more carefull. Thanks for advice.